docs/src/main/asciidoc/rest.adoc

```diff
@@ -2310,9 +2310,9 @@ quarkus.log.category."org.jboss.resteasy.reactive.common.core.AbstractResteasyRe
 
 ==== Via annotations
 
-You can declare functions which are invoked in the following phases of the request processing:
+You can declare functions that are invoked in the following phases of the request processing:
 
-- Before the endpoint method is identified: pre-routing request filter
+- Before the endpoint method is identified: pre-matching request filter
```

Comment:
@geoand can you check this one?

The rest looks good.

Reply:
Yeah, this makes sense as it's the same terminology used in the filter declaration